### DISCUSSION TITLE
Songs are sorted randomly on album play

### DISCUSSION THREAD

**Discussion by archer3cl** [jeffvli/feishin#751] [Q&A] [answered]
As title states, songs are sorted randomly on album play when playing an album from anywhere except inside the album.
My specs are Mac Feishin v9.0.0 connected to a navidrome server.

I tried to replicate this behavior over https://feishin.vercel.app/ but it seems to be only happening on my machine.

**Comment by archer3cl** [comment]
It seems it also happens when adding to a playlist and not just when playing.

**Comment by kgarner7** [comment, marked as answer]
Navidrome sanitized sorts, so only one field is possible now (see https://github.com/navidrome/navidrome/commit/3107170afd9f557a10f7031f23cb3c9e975a71f9). Workaround https://github.com/jeffvli/feishin/commit/6eecc3c0fda68c41d86d06f616d82a49d41083ea, which sorts albums/album artists (with no limit) after the fact.
